components/automate-cs-ocid/habitat/config/scripts/set_file_permissions.sh

```diff
@@ -0,0 +1,7 @@
+# Giving read+write permission to hab user
+# This is required so that 'db:migrate' can be executed from the run hook
+chmod 746 "$(hab pkg path 'chef/oc_id')/oc_id/db/schema.rb"
+chmod 746 "$(hab pkg path 'chef/oc_id')/oc_id/config/private-chef-secrets.json"
+
+# Giving read+write+execute permission to hab user on settings folder and its contents
+chmod 747 -R "$(hab pkg path 'chef/oc_id')/oc_id/config/settings"
```

Comment:
why the whole settings folder permission needs to be changed?

Reply:
We have only environment specific yml files. As we are planning to apply the configuration as per the environment so for all the files we need write permission.